mappings/net/minecraft/block/AnvilBlock.mapping

```diff
@@ -11,8 +11,15 @@ CLASS bfz net/minecraft/block/AnvilBlock
 	METHOD <init> (Lbgs$c;)V
 		ARG 1 settings
 		ARG 2 stateBuilder
+	METHOD a configureEntity (Laox;)V
+		ARG 1 entity
 	METHOD a getPlacementState (Laus;)Lbpm;
 		ARG 1 ctx
+	METHOD a (Lbbp;Let;)V
+		ARG 2 pos
+	METHOD a (Lbbp;Let;Lbpm;Lbpm;)V
+		ARG 2 pos
+		ARG 3 state
```

Comment:
There are two state parameters here, what's the difference? The names should reflect that

Reply:
I've accidentally mapped that state parameter, it's supposed to be unmapped for this PR.

Reply:
Fixed.